### ISSUE TITLE
mjob should expressly list out allowed sizes for memory

### ISSUE THREAD

**Issue by qdzlug** [joyent/node-manta#279]
Right now the help shows..

```
    --memory=MEMORY_MB                  amount of memory available for all
                                        phases (megabytes)
```

But, you get an error if you're not in a narrow range of values....

```
mlogin --memory=1232123
 * ERROR: property "phases[0].memory": does not have a value in the enumeration 128, 256, 512, 1024, 2048, 4096, 8192
 * cleaning up resources...
```

Was thinking that actually listing those values in the help would be...helpful.

Thanks!

Jay

**Comment by trentm** [comment]
I'm not sure we reliably can. The supported values I *think* is Manta deployment-specific. I.e. a separate Manta standup could be setup to support a different range of values. It would either need to be an endpoint to get that info.

Alternatively we could definitely phrase the help language better for this.

**Comment by qdzlug** [comment]
Makes sense; I will dig and see if we have an endpoint to provide that information; at that point it becomes easier to phrase the help by telling the end user where to look.

**Comment by askfongjojo** [comment]
I think the values are hardcoded currently, not really that configurable by operator.
https://github.com/joyent/manta-marlin/blob/master/common/lib/schema.js

They are also documented in the apidocs. https://apidocs.joyent.com/manta/api.html#CreateJob.
See also https://devhub.joyent.com/jira/browse/MANTA-2429 on a similar issue.

**Comment by trentm** [comment]
@askfongjojo Thanks! So Perhaps the help output for 'mjob' could also hardcode showing those values.

**Comment by trentm** [comment]
@davepacheco  Did you have an opinion on hardcoding those values in the '--memory' option help? Or perhaps a link to canonical Manta docs on those memory options?

**Comment by davepacheco** [comment]
I like the idea of linking to the canonical docs for it.  I'd suggest https://apidocs.joyent.com/manta/jobs-reference.html#resource-usage-memory-and-disk-properties.

**Comment by trentm** [comment]
https://cr.joyent.us/#/c/836/

**Comment by trentm** [comment]
Fixed in commit 67668734f7c5533c24bda1e9f8a5c8d098b21dc1
